addons/web/static/src/webclient/commands/command_service.js

```diff
@@ -27,9 +27,9 @@ export const commandService = {
         const registeredCommands = new Map();
         let nextToken = 0;
         let isPaletteOpened = false;
+        const overlayModifier = registry.category("services").get("hotkey").overlayModifier;
```

Comment:
Could you please move this closer to usage in openPalette method?

This will allow to change modifier without page reload. And will help to avoid possible problems in services start / patching sequence.

Reply:
Good idea